Source/JavaScriptCore/llint/LLIntThunks.cpp

```diff
@@ -687,7 +687,7 @@ MacroAssemblerCodeRef<NativeToJITGatePtrTag> untagGateThunk(void* pointer)
     jit.untagPtr(GPRInfo::regT3, ARM64Registers::lr);
     jit.validateUntaggedPtr(ARM64Registers::lr, GPRInfo::regT3);
     jit.move(CCallHelpers::TrustedImmPtr(pointer), GPRInfo::regT3);
-    jit.farJump(GPRInfo::regT3, OperationPtrTag);
+    jit.call(GPRInfo::regT3, OperationPtrTag);
```

Comment:
I have no idea what this function is supposed to do; I switched out the farJump for a call just to match my change up above.